src/flap.sol

```diff
@@ -49,24 +49,24 @@ contract Flapper is LibNote {
 
     // --- Data ---
     struct Bid {
-        uint256 bid;
-        uint256 lot;
+        uint256 bid;  // gems paid     [wad]
+        uint256 lot;  // dai for sale  [rad]
         address guy;  // high bidder
         uint48  tic;  // expiry time
         uint48  end;
     }
 
     mapping (uint => Bid) public bids;
 
-    VatLike  public   vat;
+    VatLike  public   vat; // CDP Engine
```

Comment:
nit: minimum of two spaces between code and start of comment

Reply:
I also sort of question whether this comment really adds much--anyone who wants to understand the system is going to have to learn what the `Vat` is pretty much immediately.

Reply:
I'm indifferent to adding short descriptions to contracts. The `Pot` originally contained short descriptions for `Vat` and `Vow` as `CDP Engine` and `Debt Engine`, so I simply persisted this pattern across the repo - I did not, however, attempt to re-describe the `Cat`, `Pot`, or `Spotty` in `End`, as they aren't used as much as the former two and would cause require more time to come to consensus on the names. The other variable names, on the other hand, have already been described in the Glossary, so it's easy to use those (or slight) variants and move forward. 

Up to you how you want to proceed - I'd be happy to delete short descriptions for contract.

Reply:
Eh, it probably isn't worth the time to remove them.